include/wget/wget.h

```diff
@@ -265,6 +265,8 @@ int
 	wget_global_get_int(int key) LIBWGET_EXPORT;
 void
 	(*wget_global_get_func(int key))(const char *, size_t) LIBWGET_EXPORT;
+void
+	wget_handle_sigwinch(void) LIBWGET_EXPORT;
```

Comment:
I am not sure if a `wget_bar_resize` style function is a better idea. When the screen size changes. we want the progress bar to automatically adapt to the new screen size. Same thing for if the height changes. If the screen height becomes less that nslots, libwget should refuse to draw the progress bar. The library should not depend on the client giving it the correct information in this case. It should pull the right information on its own.

We can however rename it to wget_bar_screen_changed and retain the current functionality, i.e.: it acts as a signal handler for SIGWINCH or any other method of detecting that the screen size has changed.